documentation/manual/map-reduce.md

```diff
@@ -48,17 +48,17 @@ The OSHDB provides the following list of default reduce operations, that are oft
 * [`estimatedQuantile(s)`](https://docs.ohsome.org/java/oshdb/0.6.1/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#estimatedQuantiles())
 * [`collect`](https://docs.ohsome.org/java/oshdb/0.6.1/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#collect())
 
-Some of the listed specialized reducers also have overloaded versions that accept a mapping function directly. This allows some queries to be written more consicely, but also allows for improved type inference: For example when summing integer values, using the overloaded [`sum`](https://docs.ohsome.org/java/oshdb/0.6.1/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#sum(org.heigit.bigspatialdata.oshdb.api.generic.function.SerializableFunction)) reducer knows that the result must also be of type `Integer`, and doesn't have to resort on returning the more generic `Number` type.
+Some listed specialized reducers also have overloaded versions that accept a mapping function directly. This allows some queries to be written more concisely, but also allows for improved type inference: For example when summing integer values, using the overloaded [`sum`](https://docs.ohsome.org/java/oshdb/0.6.1/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#sum(org.heigit.bigspatialdata.oshdb.api.generic.function.SerializableFunction)) reducer knows that the result must also be of type `Integer`, and doesn't have to resort on returning the more generic `Number` type.
 
 stream
 ------
 
-Instead of using a regular reduce operation at the end of an OSHDB query, one can also call [`stream`](https://docs.ohsome.org/java/oshdb/0.6.1/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#stream()), which doesn't aggregate the values into a final result, but rather returns a (potentially long) stream of values. If possible, using a reduce operation instead of streaming all values and using post-processing results in better performance of a query, because there is less data to be transferred. The stream operation is however beneficiall over using `collect` if the result set is expected to be large, because it doesn't require all the data to be buffered into a result collection.
+Instead of using a regular reduce operation at the end of an OSHDB query, one can also call [`stream`](https://docs.ohsome.org/java/oshdb/0.6.1/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#stream()), which doesn't aggregate the values into a final result, but rather returns a (potentially long) stream of values. If possible, using a reduce operation instead of streaming all values and using post-processing results in better performance of a query, because there is fewer data to be transferred. The stream operation is however beneficial over using `collect` if the result set is expected to be large, because it doesn't require all the data to be buffered into a result collection.
```

Comment:
I think _less data_ is used more commonly, isn't it? Especially when thinking of a stream of data, I would argue that thinking of the amount of data as an uncountable "thing" would be more fitting.
PS: Otherwise one would need to write: “_…, because there are fewer data to be transferred.”, right?

Reply:
correct, because _data_ is uncountable